src/components/notifications/AppExplorer/AppCard/index.tsx

```diff
@@ -99,6 +93,12 @@ const AppCard: React.FC<AppCardProps> = ({
     }
   }
 
+  useEffect(() => {
+    if (subscribing && subscribed) {
+      showSuccessMessageToast(`Subscribed to ${name}`)
```

Comment:
@devceline Should we call the `setSubscribing(false)` in this block as well? Seems like we don't set it to `false` if no error occurs.

Reply:
I don't think it matters because `subscribed` would be true

Reply:
Lets double make sure it's not gonna retriggered